packages/smart-contracts/test/contracts/ERC20SwapToConversion.test.ts

```diff
@@ -116,7 +116,7 @@ describe('contract: ERC20SwapToConversion', () => {
         referenceExample,
         fiatDecimal.mul(1),
         builder,
-        exchangeRateOrigin + 100, // _uniswapDeadline
+        exchangeRateOrigin + 1000, // _uniswapDeadline. 100 -> 1000: Too low value may lead to error (network dependent)
```

Comment:
Is it still mandatory ? If yes, no need to explicit the reason in a comment

Reply:
Yes it is for L2